lint/linter.go

```diff
@@ -152,7 +152,11 @@ func (l *Linter) lintPackage(filenames []string, gover *goversion.Version, ruleS
 		return nil
 	}
 
-	pkg.lint(ruleSet, config, failures)
+	err := pkg.lint(ruleSet, config, failures)
+
```

Comment:
nit: I'd remove this line (it will be more consistent stylistically).

Reply:
@denisvmedia maybe something like this: ```return pkg.lint(ruleSet, config, failures)```?